test/index.ts

```diff
@@ -1,9 +1,36 @@
 import {assert} from 'chai'
 
+import sinon from 'sinon'
+
 import * as lib from '$lib'
 
-suite('index', function () {
-    test('maths', function () {
-        assert.equal(lib.maths(1, 2), 3)
+suite('MnemonicSeed', function () {
+    const words = 'east member flee syrup naive uncover feature aim pitch sight awful weather';
+
+    suiteSetup(function () {
+        // eslint-disable-next-line @typescript-eslint/no-var-requires
+        sinon.stub(require('bip39'), 'generateMnemonic').returns(words)
+    })
+
+    test('generate', function () {
+        const mnemonicSeed = lib.MnemonicSeed.generate();
+
+        assert.equal(mnemonicSeed.words, words)
+    })
+
+    suite('derivePrivateKey', function () {
```

Comment:
Add a test that verifies that seeds generated by a ledger maps to the public key provided by the ledger eos app

Reply:
Hmm, not sure I am following. Are you saying that you want me to generate a public key using the seed (something like [this](https://github.com/iancoleman/bip39/blob/23f393acfba3f8697f42249965f4acfebae918a2/src/js/eos-util.js#L1-L7)) and compare it to the public key that is returned by core?

Reply:
Just added a test for this 👍